uat/testing-features/pom.xml

```diff
@@ -17,6 +17,8 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <skipTests>false</skipTests>
         <lombok.version>1.18.22</lombok.version>
+        <junit.version>5.9.2</junit.version>
```

Comment:
It must be part of root pom.xml when pom.xml structure will be converted from standalone to parent-children like [here ](https://github.com/aws-greengrass/aws-greengrass-client-device-auth/pull/249/files)

Reply:
Would you like it to be fixed in scope of this feature or it is better to create separate task to reorganize pom structure?

Reply:
I will extract part of other PR with reorganization.

Reply:
Merged with uat-dev 16dfc677f0b4a9523eca048ce58520b7587e2aec